main.py

```diff
@@ -1,21 +1,26 @@
 from board import *
+
+
 class App(tk.Tk):
     def __init__(self, *args, **kwargs):
         tk.Tk.__init__(self, *args, **kwargs)
-        self.canvas = tk.Canvas( self, width=640, height=640, borderwidth=0, highlightthickness=0)
+        self.canvas = tk.Canvas(
+            self, width=640, height=640, borderwidth=0, highlightthickness=0)
         self.canvas.pack(side="top", fill="both", expand="true")
         self.Game = Game()
-        self.dimensions = 8
+        self.row = 8
```

Comment:
Please make sure to use this property in line 22

Reply:
done